packages/pyright-internal/src/analyzer/typeEvaluator.ts

```diff
@@ -15114,6 +15114,10 @@ export function createTypeEvaluator(
         return { type, isIncomplete, typeErrors };
     }
 
+    function _ignoreUntypedModule(ruleset: DiagnosticRuleSet, module: string) {
+        return ruleset.allowedUntypedLibraries.some(x => (module + ".").startsWith(x + "."));
```

Comment:
since this logic is repeated, perhaps it can be abstracted into a utility function located in `./packages/pyright-internal/src/analyzer/importStatementUtils.ts` or something?

Reply:
fixed, put into pythonPathUtils (yeah I know it's modules not paths but I think the best place)